3.11/aql/execution-and-performance-optimizer.md

```diff
@@ -717,6 +721,36 @@ The following optimizer rules may appear in the `rules` attribute of
   case, no AQL is executed on the DB-Servers. Instead, the Coordinator
   directly works with the documents on the DB-Servers.
 
+- `optimize-cluster-multiple-document-operations`:
+  Bulk `INSERT` operations in cluster deployments can be optimized to avoid
+  unnecessary overhead that AQL queries typically require for the setup and
+  shutdown in clusters.
```

Comment:
```suggestion
  Bulk `INSERT` operations in cluster deployments can be optimized to avoid
  unnecessary overhead that AQL queries typically require for the setup and
  shutdown in clusters, as well as for the internal batching.
```